pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -387,6 +387,8 @@ class Build {
                         }
 
                         def additionalTestLabel = buildConfig.ADDITIONAL_TEST_LABEL
+                        def relatedNodeLabel = buildConfig.RELATED_NODES
```

Comment:
I do not think we have buildConfig.RELATED_NODES defined. The default relatedNodeLabel should be an empty string.

Reply:
Thanks for pointing out. Updated.